scripts/ansible_setup/prepare.yml

```diff
@@ -0,0 +1,138 @@
+- name: Uninstall old versions
```

Comment:
This file should not be committed anymore since we changed the ansible scripts

Reply:
Thanks @AlexAntn, removed it